docs/other-inverters.md

```diff
@@ -58,3 +58,88 @@ Discussion ticket is here: <https://github.com/springfall2008/batpred/issues/684
 
 - Please copy the template apps.yaml from <https://github.com/springfall2008/batpred/blob/main/templates/huawei.yaml> and modify for your system
 - Ensure you set **input_number.predbat_set_reserve_min** to the minimum value for your system which maybe 12%
+
+## Solar edge Inverters
+
+Discussion ticket is here: <https://github.com/springfall2008/batpred/issues/181>
+
+- Please copy the template apps.yaml from <https://github.com/springfall2008/batpred/blob/main/templates/solaredge.yaml> and modify for your system
+- Ensure that **number.solaredge_i1_storage_command_timeout** is set to reasonably high value e.g. 3600 seconds to avoid the commands issued being cancelled
+- Power Control Options, as well as Enable Battery Control, must be enabled in the Solaredge Modbus Multi integration configuration, and switch.solaredge_i1_advanced_power_control must be on.
+
+- For the pv_today sensor to work you need to create this as a template, please see: <https://gist.github.com/Ashpork/f80fb0d3cb22356a12ed24734065061c>.
+This sensor isn't critical so you can just comment it out in apps.yaml if you can't get it to work
+
+```yaml
+template:
+  - sensor:
+      - name: "Solar Panel Production W"
+        unique_id: solar_panel_production_w
+        unit_of_measurement: "W"
+        icon: mdi:solar-power
+        state: >
+          {% set i1_dc_power = states('sensor.solaredge_i1_dc_power') | float(0) %}
+          {% set b1_dc_power = states('sensor.solaredge_b1_dc_power') | float(0) %}
+          {% if (i1_dc_power + b1_dc_power <= 0) %}
+            0
+          {% else %}
+            {{ (i1_dc_power + b1_dc_power) }}
+          {% endif %}
+        availability: >
+          {{ states('sensor.solaredge_i1_dc_power') | is_number and states('sensor.solaredge_b1_dc_power') | is_number }}
+sensor:
+  - platform: integration
+    source: sensor.solar_panel_production_w
+    method: left
+    unit_prefix: k
+    name: solar_panel_production_kwh
+```
+
+## I want to add my newer inverter to Predbat
```

Comment:
Maybe this should be worded as "I want to add my unsupported inverter", seeing as it's not necessarily just newer inverters that are presently not supported?